virttest/env_process.py

```diff
@@ -1240,6 +1240,25 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio-win version, if not satisfied, cancel test
+    if params.get("required_virtio_win"):
+        if params.get("cdrom_virtio"):
+            required_virtio_win = params.get("required_virtio_win")
+            LOG.info("Test requires virtio-win version: %s" % required_virtio_win)
+            cdrom_virtio = params["cdrom_virtio"]
+            if re.search('prewhql', cdrom_virtio):
+                match = re.search("\d+.\d+-\d+", cdrom_virtio)
+            else:
+                match = re.search("\d+.\d+\d+-\d+", cdrom_virtio)
+            if match is None:
+                test.cancel("Can not get virtio-win version")
+            cdrom_virtio_version = re.sub('-', '.', match.group(0))
```

Comment:
Could you add comment with the example of the format to define `required_virtio_win`, maybe also including other related information like how virtio_win version strings usually look like, maybe within "base.cfg" or somewhere?

Reply:
How about add a annotation here like that:
`# Example value of required_virtio_win: [0.1.180, 0.1.280] or [1.9.8.6,1.9.35.0)`

Reply:
@leidwang I suppose a better location would be at .cfg configuration file: `# required_virtio_win = [0.1.180, 0.1.280]`, which aligns to most of param definition examples given in this project.